src/init/starship.ps1

```diff
@@ -63,6 +63,11 @@ function global:prompt {
         $process.StandardOutput.ReadToEnd();
     }
 
+    # Invoke precmd, if specified
+    if (Test-Path ENV:starship_precmd_user_func) { 
+        Invoke-Expression $ENV:starship_precmd_user_func
```

Comment:
Can you change this to use a function instead of a string variable?

Reply:
@davidkna done... didn't know I could test for a given function name... It's now set to call `Invoke-Starship-PreCommand`  if it exists.